src/NuGet.Protocol.Catalog/Models/AlternatePackage.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class AlternatePackage
```

Comment:
`AlternatePackage` was intended to be identical to `PackageDependency`, so you can replace this class with that.

If you would prefer not to do that, we should at least add `[JsonConverter(typeof(PackageDependencyRangeConverter))]` to `Range`.

Reply:
`PackageDependencyRangeConverter` is for quirks. Do we have quirks in `AlternatePackage`? My understanding was that gallery validated the package version range.

Reply:
Oh, there shouldn't be any quirks. Gallery doesn't accept any version range strings--it either supports a min version (`[1.0.0, )`) or any version (`(, )`). Db2Catalog creates these ranges itself.

It's possible that we may misalign `AlternatePackage` with `PackageDependency`, but I think there should be some link between them. Maybe a base class (`PackageReference`) that `AlternatePackage` and `PackageDependency` extend?

Reply:
I have responded to your latest comment here: https://github.com/NuGet/NuGet.Services.Metadata/pull/691#discussion_r343277774.

Reply:
The PackageDependency and the AlternatePackage are very similar. Maybe add some comments what the difference between them? 
Do they need to be somehow in sync ? If yes I think that @scottbommarito 's suggestion is good.

Reply:
@cristinamanum, we have separated the notions of package dependency and alternate package.